canary/environments/pull-environments.sh

```diff
@@ -42,12 +42,9 @@ for dir in ./environments/*/ ; do
   cd $dir
   echo $dir
 
-  # 'echo y' is used to answer "Yes" to the prompt "Do you plan on modifying this backend?"
+  # 'echo n' is used to answer "No" to the prompt "Do you plan on modifying this backend?"
   # See https://github.com/aws-amplify/amplify-cli/issues/5275
-  echo y | yarn --cwd environments/ pull \
-    --amplify $AMPLIFY \
-    --frontend $FRONTEND \
-    --providers $PROVIDERS
+  echo n | yarn pull --amplify $AMPLIFY --frontend $FRONTEND --providers $PROVIDERS
```

Comment:
I've changed y to n, because I found pulling problems (workflow) when using y with IAM roles.

Haven't looked too deep into it, but since we don't plan on modifying on backend in CI anyway, I'm not concerned with changing this to n.